main.nf

```diff
@@ -436,13 +436,8 @@ workflow waspRealigning {
 			| align_reads
 			| join(dedup_bam, by: [0, 1])
 			| wasp_filter_reads
-			| map(it -> tuple(it[0], tuple(it[1], true)))
 		
-		merged_out_bam = nodata	
-			| mix(filtered_bam)
-			| groupTuple(size: 2)
-			| filter_grouped_channel
-			| merge_bam_files
+		merged_out_bam = merge_bam_files(filtered_bam)
```

Comment:
Had to change this part because otherwise merge_bam_files, count_reads and  reformat2babachi weren't starting, but main.nf script was running totally fine. @wishabc if you have any ideas why it is like that, please, share, because I'm utterly perplexed.